deploy/kubernetes/helm/che/custom-charts/che-devfile-registry/templates/configmap.yaml

```diff
@@ -23,3 +23,4 @@ data:
   CHE_DEVFILE_IMAGES_REGISTRY_TAG: {{ . | quote }}
   {{- end }}
 {{- end }}
+  CHE_DEVFILE_REGISTRY_URL: {{ template "devfileRegistryUrl" . }}
```

Comment:
We need something like this here https://github.com/eclipse-che/che-server/blob/main/deploy/kubernetes/helm/che/templates/configmap.yaml#L122-L132
```
{{- if .Values.che.workspace.devfileRegistryUrl }}
  CHE_WORKSPACE_DEVFILE__REGISTRY__URL: {{ .Values.che.workspace.devfileRegistryUrl | quote }}
{{- else }}
  CHE_WORKSPACE_DEVFILE__REGISTRY__URL: {{ template "devfileRegistryUrl" . }}
{{- end }}
```

Reply:
fixed

Reply:
to check if there is no issues with templates, pls use

```
cd <CHE_SERVER_REPOSITORY>/deploy/kubernetes/helm/che
helm dependency update
chectl server:deploy --platform minikube --installer helm -m --templates <CHE_SERVER_REPOSITORY>/deploy/
```